### ISSUE TITLE
Doubts about luminoth

### ISSUE THREAD

**Issue by AshwinAce** [tryolabs/luminoth#249]
I have some doubts about luminoth. I thank you for the official documentation released a few weeks back. I have completed training on my dataset. I have a doubt about the transform and eval commands.

Before training, I transformed train and valids alone. Is it possible to transform multiple datasets as test? I want to run eval on two different pieces of data. One possibility I can think of is to transform, run eval and then do it again, replacing the test folder tfrecords with the new set's tfrecords.

For eval, I see that you can plot loss metrics against number of epochs. In my case, training is finished. When I run eval now, will I be able to get these plots? I guess not because I think I have to run both in parallel for a plot against time.

Is there any place where the training losses have been stored or are they gone now?

I see that we get mean Average Precision metrics after running an eval command. Are there are further outputs that you get after this step?
